processor/authenticationprocessor/README.md

```diff
@@ -0,0 +1,23 @@
+# Authentication Processor
+
+* Supported pipeline types: traces
+* Status: in development
+
+This processor authenticates the incoming traces by extracting the authentication data from the context
+and verifying the bearer token with the specified provider.
+
+Currently, only bearer token authentication is supported, added as part of `PerRPC` gRPC authentication.
+It requires the gRPC client to send a header named `authorization` in line with the equivalent HTTP/2 header.
```

Comment:
Can we have the header name as a config option with a default value `"authorization"`?